install/configuring_inventory_file.adoc

```diff
@@ -1477,6 +1470,13 @@ use persistent storage for metrics. If
 is stored in an `emptyDir` volume, which will be deleted when the Cassandra pod
 terminates.
 
+[IMPORTANT]
```

Comment:
The verbiage here can likely be more specific. 
Based on: http://cassandra.apache.org/doc/latest/operating/hardware.html#disks 

Suggested wording change: 
```
Cassandra is designed to provide redundancy via multiple independent, instances. For this reason, using NFS or a SAN for data directories is an antipattern and should typically be avoided.

However, NFS/SAN implementations on the marketplace may not have issues backing or providing storage to this component. Please contact the individual NFS/SAN implementation vendor for more information on any testing they may have performed against these OpenShift framework components.
```

Reply:
I added this in with the existing content + minor tweaks in the new PR.